opm/input/eclipse/Schedule/BCProp.hpp

```diff
@@ -76,6 +91,7 @@ class BCProp {
         {
             serializer(index);
             serializer(bctype);
+            serializer(bcmechtype);
```

Comment:
I think also mechbcvalue must be serialized. That is the only remaining problem I see in the PR.

Reply:
This is still missing.